markdown/ch6.markdown

```diff
@@ -359,41 +372,105 @@ x = Dinv2 @ eigvecs[:, 1]
 ```
 
 (The reasons for using this vector are too long to explain here, but appear in
-the paper's supplementary material, linked above.)
+the paper's supplementary material, linked above. The short version is that
+choosing this vector minimizes the total length of the links between neurons.)
+
+There is one small kink that we must address before proceeding: eigenvectors
+are only defined up to a multiplicative constant. This follows simply from the
+definition of an eigenvector: suppose $v$ is an eigenvector of the matrix $M$,
+with corresponding eigenvalue $\lambda$. Then $\alpha v$ is also an eigenvector
+of $M$ for any scalar number $\alpha$, because
+$M(\alpha v) = \alpha Mv = \alpha \lambda v = \lambda (\alpha v)$. So, it is
+arbitrary whether a software package returns $v$ or $-v$ when asked for the
+eigenvectors of $M$. In order to make sure we reproduce the layout from the
+Varshney et al paper, we must make sure that the vector is pointing in the same
+direction, rather than the opposite direction. We do this by choosing an
+arbitrary neuron from their Figure 2, and checking the sign of `x` at that
+position. We then reverse it if it doesn't match its sign in Figure 2 of the
+paper.
+
+```python
+vc2_index = np.flatnonzero(neuron_ids == 'VC02')[0]
```

Comment:
I don't understand the question. Are you saying we should make a hubbub of an introduction of flatnonzero here?